main.cpp

```diff
@@ -242,6 +257,17 @@ float Laptop::saveData(int storage)
     
     return(hardDriveAvailableGb);
 }
+
+struct LaptopWrapper
+{
+    LaptopWrapper(Laptop* LaptopPointer) : laptopptr(LaptopPointer){}
+    ~LaptopWrapper()
+    {
+        delete laptopptr;
+    }
+    
+    Laptop* laptopptr = nullptr;
```

Comment:
`camelCase` your member variable name.  
Type names begin with capital letters.  variables do not.
- [ ] revise